recipes/noiseflow/build_linux.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+git clone https://github.com/xtensor-stack/xtensor-fftw extern/xtensor-fftw
+git clone https://github.com/kfrlib/kfr extern/kfr
```

Comment:
If these clones are to provide new dependencies, you'll need to provide these as conda-forge packages instead

Reply:
Thanks for your mention. Those 2 packages worked well, I have provided all dependencies for them.

Reply:
What @jaimergp means : there should not be a git clone here. Rather you should create (in the PR) two additional feedstocks (one for each dependency). 

Now. `xtensor-fftw` is simply available: https://github.com/conda-forge/xtensor-fftw-feedstock , so no need to clone or to create a new feedstock. Just add `xtensor-fftw` to `build`.

Reply:
In addition, conda-build seems to fail over this

Reply:
For conda build have be failed over there. Nope, you are looking for the new commit for same PR, which I have made some changes according to @jaimergp's suggestions to do the tests. In fact, they are all working well, I can literally install the package but just can't import it in the test stage

Reply:
Yes!

Reply:
Hey, sorry for the late reply, I lost my glasses 3 days ago, and now I am back to work. I think I need to use `git clone`, because I need to use `FindFFTW.cmake` script from `xtensor-fftw` in my cmake file. An easy approach is to just copy the `xtensor-fftw` repositories to my package, but I don't want to do that. So I use `git clone` to make everybody know that it's from `xtensor-fftw` repositories.

Reply:
And finally i solved the problem, it's `poetry` problem. I must add the following lines in my `pyproject.toml` to let the building tool to copy the `*.py` file to package. And in `macos` and `win`, there are no problems when I didn't add those lines, but in `linux` (at least in `centos`, cuz conda-forge use the docker of `centos`) I must add those lines.

Thanks for your guys help!!! @jaimergp @tdegeus 

```toml
[tool.poetry]
include = ["CMakeLists.txt", ".gitmodules", "noiseflow/**/*.py"]
exclude = ["noiseflow/tests/**/*"]
```